src/components/NewMovie/NewMovie.js

```diff
@@ -1,13 +1,140 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
 
 export class NewMovie extends Component {
-  state = {};
+  state = {
+    title: '',
+    description: '',
+    imgUrl: '',
+    imdbUrl: '',
+
+    errorTitle: false,
+    errorDesc: false,
+    errorImg: false,
+    errorImdb: false,
+  };
+
+  SubmitCheck = (event) => {
```

Comment:
Let's follow JS naming convention.

```suggestion
  submitCheck = (event) => {
```